js/client/bootstrap/modules/internal.js

```diff
@@ -64,6 +64,33 @@
     delete global.SYS_INTERRUPT_TO_DEADLINE;
   }
 
+  // //////////////////////////////////////////////////////////////////////////////
+  // / @brief monitor a subprocess
+  // //////////////////////////////////////////////////////////////////////////////
+
+  if (global.SYS_ADD_TO_PID_MONITORING) {
+    exports.addPidToMonitor = global.SYS_ADD_TO_PID_MONITORING;
+    delete global.SYS_ADD_TO_PID_MONITORING;
+  }
+
+  // //////////////////////////////////////////////////////////////////////////////
+  // / @brief unmonitor a sub process
+  // //////////////////////////////////////////////////////////////////////////////
+
+  if (global.SYS_REMOVE_FROM_PID_MONITORING) {
+    exports.removePidFromMonitor = global.SYS_REMOVE_FROM_PID_MONITORING;
+    delete global.SYS_REMOVE_FROM_PID_MONITORING;
+  }
```

Comment:
Just wondering why it is `SYS_ADD_TO_PID_MONITORING` and not `SYS_ADD_PID_TO_MONITORING`, and why it is `SYS_REMOVE_FROM_PID_MONITORING` and not `SYS_REMOVE_PID_FROM_MONITORING`?

Reply:
for me it it was `PidMonitoring` to which an entry was enlisted or removed; Grammer in variable names is non-existent ;)